.github/workflows/build.yml

```diff
@@ -40,177 +37,249 @@ jobs:
+              "/p:AndroidSigningKeyStore=$("app_fdroid-keystore.jks")" `
+              "/p:AndroidSigningStorePass=$($env:FDROID_KEYSTORE_PASSWORD)" "/v:quiet"
 
-      - name: Set up F-Droid server
-        if: github.event_name == 'release'
-        run: |
-          sudo apt-get -qq update
-          sudo apt-get -qqy install --no-install-recommends fdroidserver wget
+          Write-Output "########################################"
+          Write-Output "##### Copy FDroid apk to project root"
+          Write-Output "########################################"
 
-      - name: Set up git credentials
-        if: github.event_name == 'release'
-        env:
-          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
-        run: |
-          git config --global credential.helper store
-          echo "https://${ACCESS_TOKEN}:x-oauth-basic@github.com" >> ~/.git-credentials
-          git config --global user.email "ci@bitwarden.com"
-          git config --global user.name "Bitwarden CI"
+          $signedApkPath = $($env:GITHUB_WORKSPACE + "/src/Android/bin/FDroid/com.x8bit.bitwarden-Signed.apk");
+          $signedApkDestPath = $($env:GITHUB_WORKSPACE + "/com.x8bit.bitwarden-fdroid.apk");
 
-      - name: Print environment
-        if: github.event_name == 'release'
-        run: |
-          node --version
-          npm --version
-          git --version
-          Write-Output "GitHub ref: $env:GITHUB_REF"
-          Write-Output "GitHub event: $env:GITHUB_EVENT"
+          Copy-Item $signedApkPath $signedApkDestPath
         shell: pwsh
-        env:
-          GITHUB_REF: ${{ github.ref }}
-          GITHUB_EVENT: ${{ github.event_name }}
 
-      - name: Checkout repo
-        if: github.event_name == 'release'
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
-
-      - name: Install Node dependencies
-        if: github.event_name == 'release'
-        run: npm install
+      - name: Upload F-Droid .apk artifact
+        uses: actions/upload-artifact@ee69f02b3dfdecd58bb31b4d133da38ba6fe3700 # v2.2.4
+        with:
+          name: com.x8bit.bitwarden-fdroid.apk
+          path: ./com.x8bit.bitwarden-fdroid.apk
 
-      - name: Decrypt secrets
-        if: github.event_name == 'release'
+      - name: Check if RC branch exists
+        id: rc-branch-check
         run: |
-          ./.github/scripts/decrypt-secret.ps1 -filename store_fdroid-keystore.jks.gpg `
-            -output ./store/fdroid/keystore.jks
-        shell: pwsh
-        env:
-          DECRYPT_FILE_PASSWORD: ${{ secrets.DECRYPT_FILE_PASSWORD }}
+          if [[ $(git ls-remote --heads origin rc) ]]; then
+            echo "::set-output name=branch_exists::1"
+          else
+            echo "::set-output name=branch_exists::0"
+          fi
+        shell: bash
 
-      - name: Compile for F-Droid Store
-        if: github.event_name == 'release'
+      - name: Deploy to Play Store
```

Comment:
It's a fairly important part of our mobile development process since we need builds in the Apple/Google test tracks before an official release takes place (for both internal and potential public beta testing).  I'm not familiar enough with GH Actions to know if there's an alternative way to kick off a deploy without a commit.